app/assets/stylesheets/application.scss

```diff
@@ -9,9 +9,30 @@
  * compiled file so the styles you add here take precedence over styles defined in any styles
  * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
  * file per style scope.
- *
- *= require_tree .
- *= require_self
- *= require foundation_and_overrides
 
+*= require_tree
+
+*= require foundation_and_overrides
+*= require_self
  */
+
+
+.side-nav-border {
+  width: 110%;
+  position: relative;
+  left: -1.1em;
+  padding-right: 20em;
+  border: 3px;
+  padding: 0.15rem 0.40rem;
+  border-width: 0 1px;
+  border-color: brown;
```

Comment:
Color `brown` should be written in hexadecimal form as `#a52a2a`<br>Color literals like `brown` should only be used in variable declarations; they should be referred to via variable everywhere else.